app/src/main/java/com/cornellappdev/transit/ui/viewmodels/HomeViewModel.kt

```diff
@@ -185,20 +183,4 @@ class HomeViewModel @Inject constructor(
         locationRepository.instantiate(context)
     }
 
-    // Map state
-
-    /**
-     * Emits whether a route should be showing on the map
-     */
-    val mapState: MutableStateFlow<MapState> =
-        MutableStateFlow(MapState(isShowing = false, routeOptionType = RouteOptionType.None))
-
-    /**
-     * Set map state for home screen
-     */
-    fun setMapState(value: MapState) {
-        mapState.value = value
-    }
-
-
```

Comment:
Seems like we're tidying up this VM, i like it! There was a LOT of stuff in here, pulling things out seems to organize things a lot.